src/controller.coffee

```diff
@@ -140,6 +140,8 @@ define ['droplet-helper',
   exports.Editor = class Editor
     constructor: (@wrapperElement, @options) ->
       @paletteGroups = @options.palette
+      @willShowPaletteWithText = @options.willShowPaletteWithText ? false
```

Comment:
Another approach might be to call it something like togglePaletteOnTransition which defaults to true.